### ISSUE TITLE
Aircraft stays in "landing" mode after full landing. Virtual stick not available

### ISSUE THREAD

**Issue by keivanh** [dji-sdk/Mobile-SDK-Android#470]
SDK ver 4.11.1
Mavic 2 Pro

After full landing by using "startLanding" in FlightController, I can't use or enable virtual stick mode anymore. "setVirtualStickModeEnabled" returns an error "Aircraft is landing". 
To clear it, either I have to turn off/on the drone or use RC sticks and move drone manually.

**Comment by dji-dev** [comment]
Public comment from Bruce Cheung in <a href='https://djisdksupport.zendesk.com/agent/tickets/31757'>Zendesk ticket #31757</a>:<br><br>Dear Customer,

Thank you for contacting DJI.

Have received the information you provided, we will forward this issue to the corresponding engineers for analysis and processing. If need more information or have the results, we will notify you as soon as possible.

Thank you for your understanding and support, hope you have a nice day.

Best Regards,
DJI SDK Support Team

**Comment by dji-dev** [comment]
Public comment from Luce Luo in <a href='https://djisdksupport.zendesk.com/agent/tickets/31757'>Zendesk ticket #31757</a>:<br><br>Dear Customer,

Thank you for contacting DJI.
It's normal that you can't enable the virtual stick when in landing status.
Please cancelLanding first if you can want to control it again.

Thanks,

Luce Luo
DJI Dev Team

**Comment by keivanh** [comment]
No, I tried that as well. First drone (Mavic 2 Pro) fully lands, I get all confirmations of landing. Then I call [(cancleLanding)](https://developer.dji.com/api-reference/android-api/Components/FlightController/DJIFlightController.html?search=cancella&i=0&#djiflightcontroller_cancelautolanding_inline) and also get no error in callback. So everything should be rested. When drone take off again I can't get virtual stick. I have to use RC joysticks and move drone manually and after few seconds virtual stick will be available again. Without moving drone by RC, it stays at "landing state" forever.

**Comment by dji-dev** [comment]
Public comment from Luce Luo in <a href='https://djisdksupport.zendesk.com/agent/tickets/31757'>Zendesk ticket #31757</a>:<br><br>Dear Customer,

Thank you for contacting DJI.
That's so weird.
Can you please give me more detail and I will report this issue to the engineer?
-Mavic2 Pro firmware version
-Probability of this issue

Thanks,

Luce Luo
DJI Dev Team
